### PR TITLE
Add Windows support to integration tests 

### PR DESCRIPTION
**Pull request by ionutbalutoiu**
## High-level description

This pull request adds Windows support for some of the current integration tests.

## Corresponding DC/OS tickets

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-2078](https://jira.mesosphere.com/browse/DCOS_OSS-2078) Integrate Hybrid DC/OS Cluster into CI/CD.

### CODE REVIEW

**Comment by ionutbalutoiu** [comment]
@mellenburg, please review this PR

File: packages/dcos-integration-test/extra/test_applications.py [hunk 1]
```diff
@@ -57,6 +57,20 @@ def deploy_test_app_and_check(dcos_api_session, app: dict, test_uuid: str):
             assert json_uid != 0, ("App running as {} should not have uid 0.".format(marathon_user))
 
 
+def deploy_test_app_and_check_windows(dcos_api_session, app: dict, test_uuid: str):
```
Comment (margaret) [inline packages/dcos-integration-test/extra/test_applications.py]: Should this also have `pytest.mark.supportedwindowsonly`?

**Comment by margaret** [comment]
Hi! This looks good to me and @sschneid, but we'd like to run it on our teamcity CI before merging. Scott's still working on our windows build job, but that should hopefully be up soon.

**Comment by ionutbalutoiu** [comment]
Sounds good. Thank-you!

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by sschneid** [comment]
Note the passing `teamcity/dcos/test/azure/hybrid-cloud` test!  😄

**Comment by sschneid** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by orsenthil** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by ionutbalutoiu** [comment]
@sschneid, @orsenthil I just rebased against current master as adviced by `mesosphere-mergebot-dev`. Everything is still green and passing.

**Comment by margaret** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by sschneid** [comment]
ACS hybrid integration tests are passing again on this PR after the most recent rebase: https://teamcity.mesosphere.io/viewLog.html?buildId=1105613&tab=buildResultsDiv&buildTypeId=DcOs_Open_Test_IntegrationTest_AcsEngineHybridCloud

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by ionutbalutoiu** [comment]
@margaret, @sschneid I just rebased the PR against current upstream master as advised @mesosphere-mergebot-dev

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by orsenthil** [comment]
@mesosphere-mergebot label work in progress

**Comment by orsenthil** [comment]
This breaks a consumer Enterprise build. We are adding a change to enterprise before we can include this in.

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by sschneid** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Pull request merged**